toolchain/sem_ir/inst_namer.cpp

```diff
@@ -569,7 +569,8 @@ auto InstNamer::CollectNamesInBlock(ScopeId top_scope_id,
       case CARBON_KIND(Call inst): {
         auto callee_function = GetCalleeFunction(*sem_ir_, inst.callee_id);
         if (!callee_function.function_id.has_value()) {
-          break;
+          add_inst_name("");
+          continue;
```

Comment:
Now you've moved the fallback code into the `default` block, how would you feel about consolidating onto using `break;` everywhere instead of using `continue;` everywhere? I think it's a bit more readable to break out of the switch than to jump immediately to the next loop iteration.

Reply:
Let me get back to you with another PR: I was wondering what it would look like to try to factor out the `switch` here, since it's a long function with nuanced flow. If I do that, it becomes `return`.

Reply:
Sounds good, thanks :)

Reply:
For reference, #5503, which it looks like was autoassigned to you (if you don't like the approach, I'll just rewrite the PR to a `break;`-ing change)